dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -118,6 +122,7 @@ public class EntryResource implements AuthenticatedResourceInterface, AliasableR
     public static final String VERSION_NOT_BELONG_TO_ENTRY_ERROR_MESSAGE = "Version does not belong to entry";
     public static final String ENTRY_NO_DOI_ERROR_MESSAGE = "Entry does not have a concept DOI associated with it";
     public static final String VERSION_NO_DOI_ERROR_MESSAGE = "Version does not have a DOI url associated with it";
+    public static final String ENTRY_NOT_DELETABLE_MESSAGE = "The specified entry is not deletable.";
```

Comment:
Wondering if we should make the error message more descriptive like "The specified entry is not deletable because it was previously published" so the user knows why they can't delete it. Alternatively, this can be handled in the UI through a disabled delete button and a descriptive tool tip

Reply:
> Wondering if we should make the error message more descriptive like "The specified entry is not deletable because it was previously published" so the user knows why they can't delete it. Alternatively, this can be handled in the UI through a disabled delete button and a descriptive tool tip

I'm not opposed to the idea. 

My plan for the UI was to include the requirements in the "confirm delete" dialog.  For now, how's about we stick with the simple message/scheme, and see how it all works, and if we need more detail in the errors, we can bolt it on later?  

(There's an interesting solution that involves "annotated" return values, aka an `AnnotatedBoolean` object that encapsulates both a boolean value and some explanatory text.  Or maybe a generic version thereof.)